src/renderers/dom/client/ReactMount.js

```diff
@@ -432,14 +431,9 @@ var ReactMount = {
       'for your app.'
     );
 
-    var nextWrappedElement = ReactElement(
+    var nextWrappedElement = ReactElement.createElement(
       TopLevelWrapper,
-      null,
-      null,
-      null,
-      null,
-      null,
-      nextElement
+      { child: nextElement }
```

Comment:
It doesn't break devtools but it does start showing the top level wrapper.

I switched this to an explicit flag on the wrapper type and updated devtools to support this new flag:

https://github.com/facebook/react-devtools/pull/407